src/relay-utils/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@adeira/relay-utils",
+  "private": false,
+  "version": "0.1.0",
+  "main": "src/index",
+  "module": "src/index.mjs",
```

Comment:
We don't write `module` manually in Universe - it's being created and added automatically when exporting to NPM.

Reply:
Should we have a scanner test for this?

Reply:
Well, technically there is nothing wrong about this. We allow writing such configuration in which case the module is not being generated automatically. But since you don't have this MJS file then it doesn't make much sense to write it here manually. So yes and no. 😬

Reply:
So the test could be if the file specified in module does not exist, the test should give an error. 

I think it could be useful.

Reply:
Yes, feel free to create it in a separated PR. :)

Reply:
Will do